src/Features/VisualBasic/Portable/Structure/Providers/MultiLineIfBlockStructureProvider.vb

```diff
@@ -10,13 +10,149 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Structure
     Friend Class MultiLineIfBlockStructureProvider
         Inherits AbstractSyntaxNodeStructureProvider(Of MultiLineIfBlockSyntax)
 
-        Protected Overrides Sub CollectBlockSpans(node As MultiLineIfBlockSyntax,
-                                                  spans As ArrayBuilder(Of BlockSpan),
-                                                  options As OptionSet,
-                                                  cancellationToken As CancellationToken)
+        Protected Overrides Sub CollectBlockSpans(node As MultiLineIfBlockSyntax, spans As ArrayBuilder(Of BlockSpan), options As OptionSet, cancellationToken As CancellationToken)
+            ' The full block: eg If ... Then ... End If
             spans.AddIfNotNull(CreateBlockSpanFromBlock(
-                               node, node.IfStatement, autoCollapse:=False,
-                               type:=BlockTypes.Conditional, isCollapsible:=True))
+                node, node.IfStatement, autoCollapse:=False,
+                type:=BlockTypes.Conditional, isCollapsible:=True))
```

Comment:
lines 13-17 do not seem like they needed to change.  You've also made hte code too long to view in github :)

Please avoid unnecessary changes that are not substantively improving the code :)